spec/controllers/agents/motifs_controller_spec.rb

```diff
@@ -66,6 +66,24 @@
         expect(response).to be_successful
       end
     end
+
+    context "with old deleted motif name" do
+      let!(:old_motif) { create(:motif, deleted_at: Time.now) }
+      let(:valid_attributes) do
+        build(:motif, name: old_motif.name).attributes
+      end
+
+      it "creates a new Motif" do
+        expect do
+          post :create, params: { organisation_id: organisation_id, motif: valid_attributes }
+        end.to change(Motif, :count).by(0)
+      end
+
```

Comment:
it "reactivate the motif" do 
        expect(old_motif.deleted_at).to be_nil
      end